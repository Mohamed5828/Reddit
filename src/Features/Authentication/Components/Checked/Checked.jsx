import { BsCheckLg } from 'react-icons/bs';
import { CheckedContainer } from './Checked.styled';

/**
 * Component to return a checked icon
 * @returns {React.Component}  Checked component
 */
 const Checked = () => {
    return (
      <CheckedContainer>
        <BsCheckLg size={18}></BsCheckLg>
      </CheckedContainer>
    );
  };
  
  export default Checked;