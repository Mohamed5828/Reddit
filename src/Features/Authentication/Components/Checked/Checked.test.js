import { shallow } from "enzyme";

import Checked from "./Checked";

describe("Checked Component", () => {
  it("this is a test for Checked Component", () => {
    expect(shallow(<Checked />)).toMatchSnapshot();
  });
});
