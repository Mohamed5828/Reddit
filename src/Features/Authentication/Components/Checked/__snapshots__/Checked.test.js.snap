// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checked Component this is a test for Checked Component 1`] = `
<styled.div>
  <BsCheckLg
    size={18}
  />
</styled.div>
`;
